Backoffice/EventSubscriber/DeleteNodeSubscriber.php

```diff
@@ -16,7 +17,9 @@ class DeleteNodeSubscriber extends AbstractDeleteSubscriber
     public function addNodeTrashCan(NodeEvent $event)
     {
         $node = $event->getNode();
-        $this->createTrashItem($node, $node->getName());
+        $name = $node->getName() . " (".$node->getLanguage()." - v".$node->getVersion().")";
```

Comment:
This ``" - v".$node->getVersion()`` is a little annoying: it works well for a french or english user, but may sound weird for another language